Transcriptomics-driven metabolic pathway analysis reveals similar alterations in lipid metabolism in mouse MASH model and human
 njury, advanced fibrosis, cirrhosis, and hepatocellular carcinoma[4, 5, 6, 7, 8, 9, 10]. The underlying mechanisms for the development and progression of the disease are complex and multifactorial, and a 'multiple-hit hypothesis' has been proposed for its prediction[11]. Increased lipid synthesis and accumulation, mainly characterized by an excess of triacylglycerectides (TGs), has been reported to occur from augmented the now biogenesis and insulin-resistant adipose tissue[12]. In turn, persistent steatosis can lead to lipotoxicity, mitochondrial dysfunction, and reactive oxygen species (ROS)-induced activation of immune cells[13, 14]. Associated pro-inflammatory signaling and metabolic imbalances further aggravate the liver, leading to apoptosis, scaring, and fibrosis.

To elucidate the underlying pathophysiological mechanisms of MASLD and MASH, appropriate experimental models need to be developed. Additionally, preclinical disease models are essential for drug discovery and testing. However, due to the multifactorial nature of steatohepatitis and its heterogeneity in humans, it is challenging to establish a preclinical model that closely resembles its pathophysiology and metabolism. Several dietary (nutritional), chemical (toxin-induced), and genetic mouse models or combinations thereof have been established[15, 16, 17]. Dietary mouse models include nutrient-deficient approaches such as methionine and choline-deficient diets (MCD), and high-fat diets (HFD). HFD-induced MASLD models with the addition of fructose and/or cholesterol (also called western diet (WD)) lead to a wide spectrum of conditions that closely resemble human MASLD pathophysiology, such as insulin resistance, liver inflammation, and fibrosis[18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 7 

### RNA sequencing

Total RNA purification from liver tissues was performed using the RNeasy* Mini kit (Qiagen). The tissues were removed from the RNA stabilization solution, weighed, and homogenized in a buffer. The liver lysate was centrifuged, and the supernatant was collected. Ethanol was added to bind the RNA to the RNeasy membrane, the contaminants were washed away and the RNA was eluted with RNAes-free water. Transcriptomics analysis was performed using bulk RNA barcoding and sequencing (BRB-seq) [40]. The RNA was reverse transcribed with specific barcoded oligo-dT primers. The samples were pooled and the cDNA was purified using the DNA clean and concentrator kit, treated with exonuclease I, and subjected to second-strand synthesis to generate double-stranded cDNA. The full-length double-stranded cDNA was purified and eluted with water. The sequencing libraries were prepared by tagmentation of full-length double-stranded cDNA, purification with DNA clean and concentrator kit, and then elution with water. Afterward, the tagmented library was amplified, and the size of fragments 200-1000 bp was selected and sequenced using the Illumina NextSeq 500 platform.

### Lipidomics analysis

Liver tissues were homogenized in 150 mM ammonium bicarbonate by a Tissue layer (MM300 Tissue Layer Mixer Mill, Retsch) with a metal bead at a speed of 25 Hz for 2.5 min. The total protein content was assessed by BCA protein assay (Bio-Rad). The homogenates were extracted by methyl tert-butyl ether/methanol (7:2) containing an internal standard (IS) mixture [41, 42]. The IS mixture contained 100 pmol of PG 17:0/17:0, 1.1 mmol Clebster1/D7, 50 pmol of PG 17:0/17:0, 50 pmol of PA 17:0/17:0, 50 pmol of PG 17:0/17:0, 50 pmol of LPC 12:0, 50 pmol of LPS 13:0, 50 pmol of LPS 17:1, 50 pmol of LPE 17:1, 200 pmol of SM 18:12/12:0, 500 pmol of PC 17:0/17:0, 200 pmol of PE 17:0/17:0, 50 pmol of CF 18:12/17:0, 100 pmol of TG 17:0/17:0/17:0, 200 pmol of CE 17:0. After extraction, 20 mL of the organic phase was transferred to the 96-well plate, and evaporated using a speed vacuum (SpeedVac Concentrator, Thermo Fisher Scientific). The dried samples were re-suspended in 40 mL of 7:5 mM ammonium acetate in chloroform/methanol/propanol (1:24,v/v/v). The lipidomics analysis was performed on Orbitrap Explora 240 mass spectrometer (Thermo Fisher Scientific) coupled with direct infusion, a TriVersa NanoMate ion source (Advin Biosciences). The 5 mL extracted sample was directly infused into the mass spectrometer using a gas pressure of 1.25 psi and a voltage of 0.95 kV. The data was acquired in both positive and negative ionization mode in a single run. The total delivery time was set at 5 min 25 s. To avoid initial spray instability, the closure delay was set at 20 s. The polarity switched from positive to negative mode 150 s after contact closure time.

For the analysis of BAs, LPLs, and MGs, the liver homogenates were mixed with IS mixture and extracted with ice-cold methanol. The IS mixture consisted of 1 uM of CA-6, CDCA-4d, DCA-4d, GGA-4d, GCDCA-4g, GCDCA-4d, GUDCA-4d, HIDCA-4g, HICCA-4d, TCA-4d, TCDCA-4d, TICDA-4d, TICDA-4d, TICDA-4d, UIDCA-4d, 4d ml of LPC 17:0, LPE 17:1, and 1 uM of LPG 17:1, MG 17:0\({}^{\circ}\). The supernatant was collected and dried under vacuum (SpeedVac Concentrator, Thermo Fisher Scientific). dried samples were re-constituted in 40 mL of mobile phases A and B (1:1) mixture. Metabolite separation and detection were performed by ultra-high performance liquid chromatography coupled to high-resolution mass spectrometry (UHPLC-HRMS) using Vanguish UHPLC-Orbitrap Explorius 240 (Thermo Fisher Scientific). Mobile phase A was 10 mM ammonium acetate plus 0.01% acetic acid in water and mobile phase B was 10 mM ammonium acetate plus 0.01% acetic acid in methanol. The chromatographic separation was performed on an ACOUITY UPLC BEH Shield RP 18 column (Waters) with a flow rate of 0.35 mL/min. The applied ionization parameters were capillary voltage \(-\)2.5 kV, vaporization and ion transfer tube temperature set up at 300 \({}^{\circ}\)C, sheath gas flow rate 55 arbitrary units (AU), auxiliary gas flow rate 10 AU, and sweep gas flow rate 0 AU. The high-resolution mass spectrometry analysis was performed in negative mode with a mass range of 300-1200 m/z, running under full MS-ddMS [2] analysis mode.

For the analysis of fatty acid oxidation and ketometabolism, the homogenates were mixed with IS and extracted with an ice-cold mixture of isopropanol acetonitrile (1:1) and 0.1% acetic acid. The IS consisted of 1.5 uM N-acetylaspartate acid-d3, 60 uM acetoacetic acid-13C2, 5 uM decanoic acid-d19, 15 uM L-glutumeric acid-d5, 2.5 uM glutard-L-carnitine-d3, 3 uM 3-hydroxybutyrate acid-d4, 0.5 uM L-kynurenne-4d, 10 uM L-lactic acid-d3, 7.5 uM L-leucine-d3, 1.9 uM L-phenylalanine-d5, 1.9 uM Paryloglutamic acid-d5, 0.5 uM serotonin-d4, 10 uM L-tryptophan-13C11,15N2, 20 uM threonine-13C6,2 and 8 uM L-valine-d8 [41]. The supernatant was collected, evaporated under vacuum (SpeedVac Concentrator, Thermo Fisher Scientific), and re-constituted in 100 uL of mobile phase A. The analysis was performed on Vanguish UHPLC-Orbitrap Q-Exactive Plus (Thermo Fisher Scientific). The mobile system contained mobile phase A, 0.1% acetic acid in aqueous, and mobile phase B, 0.1% acetic acid in acetonitrile isopropanol (1:1), under the flow rate of 0.35 mL/min. ACOUITY UPLC BEH C8 column (Waters) was used for the chromatographic separation. The high-resolution mass spectrometry analysis was performed in both positive and negative ionization mode, with a mass range of 85-600 \(m\)/z, running under full MS-ddMS [2] analysis mode. The ionization parameters were as follows: capillary voltage +4.0 and -2.5 kV, heater and capillary temperature set up at 350 \({}^{\circ}\)C, sheath gas flow rate 45 AU, auxiliary gas flow rate 15 AU and sweep gas flow rate 1 AU.

Xcalibur software 4.4 (ThermoFisher Scientific) was used for data acquisition and spectrum preview. MS chromatograms of intact lipids were extracted by LipidXplorer software 1.2.8.1\({}^{\circ}\). TraceFinder software 5.1 (ThermoFisher Scientific) was used for the analysis of BAs, LPLs, MGs, fatty acid oxidation, and ketometabolism.

### Respiration assay in isolated liver mitochondria

Oxygen flux was measured in liver homogenates by respiratory (Oxygen-2k-Orboros Instruments, Innsbruck, Austria). The 500 ug of homogenate was added to 2 mL of respiration buffer (110 mM sucrose, 60 mM K +-lactobionate, 0.5 mM EGTA, 3 mM MgCl, 20 mM taurine, 10 mM KH2PO4, 20 mM HEPES (pH 7.1), at 37 \({}^{\circ}\)C). Oxidative phosphorylation was estimated with complex I (pyruvate 5 mM, malate 2 mM, glutamate 5 mM) and complex II (succinate 10 mM) substrates in the presence of ADP (2.5 mM). leak respiration was recorded after the addition of oligomycin (2.5 uM). For maximum uncoupled respiration, the proto-ophore carbonyl cyanide m-chlorophenyl hydrazone (CCCP) was titrated in 0.5 uM increments until maximal stimulation of respiration. The protocol was terminated by assessing non-mitochondrial respiration with the complex I and III inhibitors, rotencone (0.5 mM), and antimycin A (2.5 mM), respectively. Finally, the activity at complex IV was recorded with the artificial substrate N,N,N'-tetramethyl-p-phenylenediamine dihydrochloride (TMPD; 0.5 mM) and ascorbic acid (2 mM), and inhibited with azide (100 mM). Respiration states were corrected for non-mitochondrial respiration, and complex IV activity was corrected for azide inhibition.

### Statistical tests

For transcriptomics, RNA-Seq count data was tested for differential gene expression between control and MASH mice samples using the DESeq2 method [40], including a Benjamini and Hochberg multiple testing correction. The adjusted \(p\)-value cutoff was set to 0.05 for further analysis. For lipidomics, statistical difference was assessed using the Wilcoxon two-sample test with false discovery rate (FDR) correction and considered statistically significant based on a threshold of 0.05. Missing values were imputed to 1/5 of the minimum sample value.

### Datasets used in the study

The present analysis included transcriptome data from 12 studies (six human and six mouse models). The analyzed datasets were either generated in-house (WD\(+\)CCI4 vs. CTRL) or available from literature, spanning a number of different conditions and interventions. We classified the datasets based on their respective contrast in MASLD/MASH vs. control, and MASLD/MASH stage X vs. baseline MASLD/MASH for the human cases,and in dietary, chemical, or genetic intervention vs. control for the mouse cases (Table 1). We performed analysis on a total number of 11 comparisons for human and 15 for mouse models.

### GEMs used in the study

In order for our results to be consistent and comparable across different organisms, we aimed to select two GEMs that are constructed on a homolog basis and are sufficiently equivalent in terms of encompassing genes, reactions, and metabolites. To this end, we selected the Human1 (version 1.12.0) and Mouse1 (version 1.3.0) metabolic models2,4. With the aim of making the two models equivalent, we reassigned certain reactions of the mouse model to other subsystems, namely: "MAR20010" to "Isolated", "MAR20007" to "Purine metabolism", "MAR20002" to "Glycyosmbiolipid biosynthesis-lacto and neolacto series", and "MAR20005",", MAR20006", and "MAR20016" to "Transport reactions". For additional consistency, we reassigned reaction "MAR09933" to "Acrylglyceride metabolism" in both models. All of the gene identifiers provided in both GEMs and all datasets were matched using groprific" supplemented by manual curation.

To facilitate readability, we grouped certain pathways together for presentation in the manuscript figures. These groups were defined as follows: Beta-oxidation of fatty acids: Beta-oxidation of branched-chain fatty acids (mitochondrial), Beta-oxidation of di-unsaturated fatty acids (n-6) (mitochondrial), Beta-oxidation of di-unsaturated fatty acids (n-6) (peroxisomal), Beta-oxidation of even-chain fatty acids (mitochondrial), Beta-oxidation of even-chain fatty acids (mitochondrial), Beta-oxidation of odd-chain fatty acids (metochondrial), Beta-oxidation of odd-chain fatty acids (peroxisomal), Beta-oxidation of phytamic acid (peroxisomal), Beta-oxidation of only-unsaturated fatty acids (mitochondrial), Beta-oxidation of unsaturated fatty acids (n-7) (mitochondrial), Beta-oxidation of unsaturated fatty acids (n-9) (metochondrial), Beta-oxidation of unsaturated fatty acids (n-9) (peroxisomal). Carnite shuttle: Carnite shuttle (cytosolic), Carnite shuttle (metochondrial), Carnite site shuttle (peroxisomal). Cholesterol biosynthesis 1 (Bloch pathway), Cholesterol biosynthesis 2, Cholesterol biosynthesis 3 (Kanddustch-Russell pathway). Fatty acid activation (cytosolic), Fatty acid activation (endoplasmic reticulum). Fatty acid biosynthesis: Fatty acid biosynthesis, Fatty acid biosynthesis (even-chain), Fatty acid biosynthesis (odd-chain), Fatty acid biosynthesis (unsaturated). Fatty acid desaturation: Fatty acid desaturation (even-chain), Fatty acid elongation: Fatty acid elongation (even-chain), Fatty acid desaturation (odd-chain). Glycosophilogly biosynthesis: Glycyosmbiolipid biosynthesis-gabolo series, Glycosophilogly biosynthesis-lacto and neolacto series. It is important to note that TDMPA calculations and scoring were performed for each pathway separately without this grouping.

### Transcriptomics-driven metabolic pathway analysis

The first step of TDMPA is the translation of relative gene expression ratios to corresponding relative reaction flux ratios, based on the definition of gene-protein-reaction (GPR) rules, as provided in the selected GEMs. Even though absolute gene expression does not directly correlate with reaction flux for all genes, the assumption that relative changes in gene expression between two conditions correlate with changes in reaction flux profiles has been shown to perform well[9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, After calculating the reaction ratios, we then score pathways (or sub-systems as they are denoted in GEMs) based on the amount of perturbed reactions they encompass. To start with, we select a cutoff value in order to define which reactions are considered perturbed and which are not. Reactions are considered perturbed if they satisfy either \(R_{i}\)\(>\)\(1+\)_cutoff_ or \(R_{i}\)\(<\)\(1/\left(1+\)_cutoff_\(\right)\), where \(R_{i}\) is the reaction flux ratio of reaction i. In order to illustrate how this selection affects the results, we performed a sensitivity analysis on the cutoff selection (Supplementary Fig. S1-- \(W\)\(+\)CCI4 mouse model). When considering all the reaction changes larger than one percent we observed changes in almost the entire metabolic network (129 out of 141 metabolic pathways affected). The number of affected pathways dropped steeply as the cutoff was increased, namely to 96 and 57, for changes larger than 50 and 100 percent, respectively. For changes larger than 200 percent the affected pathways were only 26. Since no clear argument can be made for the choice of a larger cutoff according to our research question, we decided to keep it as low as one percent to ensure that we do not exclude any potentially valuable information.

Finally, the Pathway Reaction Score (PRS) is calculated as the percentage of perturbed reactions in the pathway, meaning the number of reactions that have an absolute computed ratio larger than the defined cutoff over the total number of reactions in the pathway. For each pathway, the PRS is calculated as:

\[\textit{PRS}=\frac{\#\textit{of perturbed reactions in pathway j}}{\#\textit{ of total reactions in pathway j}}\] (1)

We additionally computed \(p\)-values for each pathway in order to evaluate their statistical significance. We used the hypergeometric test, which is based on the hypergeometric distribution, and describes the discreet probability of k successes in m random draws without replacement, from a population of total size M that contains K objects with that attribute. In our case, and for a single pathway i, M is the total number of reactions in all pathways of the GEM, K is the number of reactions in pathway i, m is the number of reactions calculated to be perturbed based on the selected cutoff, and k is the subset of m that belongs to pathway i. The probability for the over-representation of each pathway is then calculated as:

\[p(k)=\frac{\left(\begin{array}{c}K\\ k\end{array}\right)\left(\begin{array}{c}M-K\\ m-k\end{array}\right)}{\left(\begin{array}{c}M\\ m\end{array}\right)}\] (2)

where \(\binom{i}{j}\) is the binomial coefficient.

The computed \(p\)-values were subjected to a FDR correction, using the Benjamini and Hochberg procedure [5].

It is usual that within one pathway some reactions are upregulated and others downregulated in disease conditions. In a number of these cases, the mean value of the pathway's reaction fold-changes is close to zero, but the pathway is still very perturbed. Thus, it is not always straightforward to define if a pathway is up- or downregulated and to avoid the loss of information. To this end, we additionally calculated the normalized Euclidean distance for each pathway from healthy (control) conditions as a measure of pathway perturbation (Supplementary Fig. S2).

In order to evaluate the similarity between datasets, we selected Cohen's kappa coefficient as a metric [3]. Cohen's kappa is a statistic that is traditionally used to evaluate inter-rater reliability for categorical scales. For this, we defined three distinct categories for each reaction flux ratio as calculated by TDMPA, specifically "increased" (\(R_{i}\)\(>\)1), "decreased" (\(R_{i}\)\(<\)1), and "unchanged" (\(R_{i}\)\(=\) 1). Using these definitions, we constructed the confusion matrix for each metabolic pathway of the GEM, using the WD \(+\)CCI4 model as later #1 and all the human datasets sequentially as raster #2, and then calculated Cohen's kappa coefficient accordingly.

### Reporting summary

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

## Results

### Histological characterization of WD\(+\)CCI4 mouse model

A vital first step in preclinical MASH studies is the selection of the most suitable animal model in order to address the specific research question. We decided to use a WD mouse model supplemented by CCI4 (WD\(+\)CCI4, see Methods) that accurately simulates the histological, immunological, and transcriptional characteristics of human MASH [7]. The model showed 50-fold increase in the lipid accumulation, lobular inflammation, 4-fold increase in fibrosis, and >90% micro- and macro-veicular steatosis in the livers of MASH mice as well as the presence of ballooned hepatocytes (Fig. 1). Liver transaminases (ALT and AST), and total cholesterol were significantly elevated in the plasma of the WD\(+\)CCI4-induced MASH model compared to the control group (Fig. 1).

Metabolic pathway alteration signatures in various stages of human MASH/MASH progression and various liver damage mouse models

In order to assess the WD\(+\)CCI4 mouse model's metabolic resemblance to human MASH and its suitability for metabolic intervention design, we examined the changes in metabolic pathways that occur in MASH. We investigated multiple publically available datasets covering various stages of progression of human MASH to MASH across the full histological range from normal liver tissue to MASH with severe fibrosis (Supplementary Fig. S3). To evaluate the changes in each pathway, we used TDMPA to calculate the changes in enzymatic reactions based on differential gene expression for all datasets (see Methods). Our results suggest that different aspects of lipid metabolism are severely altered throughout the various stages of the disease. To assess the similarity of the WD\(+\)CCI4 mouse model to human MASH, we calculated Cohen's kappa coefficient [3] for each pathway and human dataset (see Methods). The WD\(+\)CCI4 model exhibits a good agreement with the human MASH vs. control datasets, especially with the human MASH vs. CTRL (2) dataset. This dataset corresponds to a fibrosis score of F1 (80% F1, range F0-F2), matching the fibrosis score of the mouse model. It has been proposed that Cohen's kappa values should be interpreted as follows: values \(\leq\) 0 as indicating no agreement, 0.01-0.20 as slight, 0.21-0.40 as fair, 0.41-0.60 as moderate, 0.61-0.80 as substantial, and 0.81-1.00 as almost perfect agreement [3]. According to this scale, 57% of metabolic pathways between human MASH and the WD\(+\)CCI4 mouse model exhibit moderate to substantial or better agreement, including most pathways related to lipid metabolism such as acyl-CoA hydrolysis, fatty acid biosynthesis and elongation, arachidonic acid, eicosanoid, leukotriene, prostaglandin, glycololipid, phospholipid, and sphingolipid metabolic pathways. Finally, 25% of the pathways exhibit slight agreement and 18% no agreement between the two datasets.

We additionally compared the WD\(+\)CCI4 model to other proposed mouse models spanning various intervention methods to induce liver damage (Supplementary Fig. S2). We categorized these models into three distinct classes of interventions, namely dietary, chemical, and genetic. The majority of them are in good agreement with each other and present upregulated pathways of phospholipids, nucleotides, keratan sulfate, and cholesterol esters. All models exhibit altered fatty acid oxidation, and most result in altered bile acid biosynthesis, and sphingolipid, leukotriene, and arachidonic acid metabolism. Cholesterol metabolism and biosynthesis are perturbedly in the most severe dietary models. Overall, the HFD, WD, streptozotocin intoxication (STZ), and GLMP knockout mouse models exhibit the least severe metabolic changes. The model resulting in the most shifts within the metabolic network is the chronic CCI4 intoxication model, highlighting the potency of CCI4 as an acute liver damage agent.

 TDMPA shows substantial alterations in lipid metabolism in WD \(+\)CC4 model and close resemblance to human MASH

As the next step in TDMPA, we scored metabolic pathways based on the number of altered metabolic reactions in each of them (see Methods). The results of TDMPA for the WD\(+\)CC4 mouse model reveal multiple metabolic pathways that are significantly altered and scored highly (Fig. 2a). Most of the top-ranked pathways are related to lipid metabolism, especially fatty acids (beta-oxidation, biosynthesis, activation, desaturation, oxidation, metabolism), bile acids (biosynthesis and recycling), cholesterol (biosynthesis, metabolism, esterification),

Fig. 1: **Effect of WD \(+\) CC4 on mouse liver.****A** Microscopy of hematoxylin and eosin (HRE)-stained liver sections showing diffuse macro-vascular steatosis, lobular inflammation, and the presence of ballooned hepatocytes in the WD \(+\)CC4 group. **B** Histological scoring of the WD\(+\)CC4 group livers, given as median values. Values in parentheses denote the range of values across all mice. Coil Red O staining comparing neutral lipid content of control and WD\(+\)CC4 group (unpaired \(t\)-test; phospholipids (biosynthesis and metabolism), TGs (biosynthesis), and the carnitine shuttle, as well as oxidative phosphorylation. We performed the same analysis for human MASH F1 vs. control (corresponding to dataset MASH vs. CTRL (2) in Table 1) and MASH progression (MASH F3 vs. MASLD) datasets (Fig. 2b, c, respectively). It is clear that the mouse model matches the human MASH vs. control results very well, but is less similar to MASH progression results. This becomes even more apparent when comparing the overlap of significantly altered pathways between the three datasets (Fig. 2d). We can discern a large overlap between altered metabolic pathways in mouse and human MASH, but the resemblance of altered metabolic pathways is less in human MASH progression compared to a mouse model. Out of the identified top-scored pathways, 54% of overall changes in metabolic pathways are similar in the human MASH and WD+CC4 mouse model. This is reduced to 11% when looking into MASLD to MASH progression. Specifically in terms of metabolic pathways, only five are commonly predicted as perturbed across all three datasets, namely fatty acid desaturation, and acylglycerides, estrogen, purine, and pyrimidine metabolism pathways. Between the mouse model and human MASH vs. control, 43 pathways are common, while 18 and 11 are only predicted in the mouse and human MASH vs. control datasets, respectively. The 43 common pathways include acyl-CoA hydrolysis, acylglycerides metabolism, bile acid biosynthesis and recycling, cholesterol biosynthesis, metabolism, and esterification, glycerolipid, glycerophospholipid, and

Fig. 2: **TDMPA results for the compared cases.****a** WJD+CC4 mouse model vs. Control, **b** human MASH vs. Control, **c** human MASH F3 vs. MASLD (MASH progression). **d** Corresponding Venn diagram of the statistically significant (FDR < 0.05) altered pathways across the three datasets. Number labels on the left panels (**a, b, c**) correspond to the pathways as indicated in the table on the right. WD +CC4: Western diet supplemented by carbon tetrachloride, MASLD: Metabolic dysfunction-associated steatotic liver, MASH: Metabolic dysfunction-associated steatotic hepatitis.

 leukotriene metabolism, glycolysis/gluconeogenesis, pyruvate metabolism, oxidative phosphorylation, estrogen metabolism, folate metabolism, purine, and pyrimidine metabolism, inositol phosphate metabolism, vitamin E metabolism, retinol metabolism, keratin sulfate degradation, the largest part of fatty acid beta-oxidation, biosynthesis, activation, and desaturation, and multiple aminoacid metabolic pathways. The mouse-only pathways include chondro sulfate degradation, tricarboxylic acid cycle and glyoxylate/dicarboxylate metabolism, sphingolipid metabolism, and parts of the carnitine shuttle, fatty acid biosynthesis, elongation, oxidation, and beta-oxidation. Similarly, the human MASH vs. control-only pathways include nicotine and nicotinamide metabolism, glycosylphosphatidylinositol (GPI)-anchor biosynthesis, ether lipid metabolism, heparan sulfate degradation, and various aminoacid metabolic pathways. The MASH progression dataset results in three metabolic pathways not predicted in neither mouse nor human MASH vs. control, namely glycosphingolipid biosynthesis-lacto and neolacto series, blood group biosynthesis, and transport reactions.

Having identified the most relevant pathways, we then investigated the changes in the enzymatic reaction level (Fig. 3). Every participating reaction in the novo acylyllyceride biosynthesis, especially TG biosynthesis, was upregulated. TG degradation was also increased, especially in chylomicrons. Similarly, most reactions in glycerophospholipid biosynthesis were upregulated. In addition, cholesterol biosynthesis as well as the formation and hydrolysis of cholesterol esters were severely perturbed. These results are in agreement with previous studies reporting increased de novo biogenesis and accumulation of TGs in MASH\({}^{\circ}\), and severely impacted cholesterol homeostasis\({}^{\circ}\)\({}^{\circ}\)\({}^{\circ}\)\({}^{\circ}\)\({}^{\circ}\). Another part of the metabolic network that showed prominent changes in MASH was mitochondrial metabolism, specifically the

Fig. 3: **Metabolic network of the most affected pathways in mouse MASH model and corresponding reaction change ratios.** Green and red colors denote a calculated increase and decrease of reaction flux, respectively. Blue triangles denote a good agreement with corresponding human data (MASH vs. CTRL (2)). Reaction change ratios are reported as log2-fold-changes. MASH Metabolic dysfunction-associated steatohepatitisitis, CTRL control, HMG-CoA 3-hydroxy-3-methylglutaryl coenzyme A, MEV mevalonate, IPP isopentenyl pyrophosphate, FPP fameysl pyrophosphate, SQL squaine, LANST Nanosterol, ZYMST zymosterol, DESMST desmosterol, CHOI cholesterol, CHOI-EST cholesterol, G3P glycerol-3-phosphate, LPA carmine shuttle, fatty acid beta-oxidation, and oxidative phosphorylation. We observed that the carnitine shuttle shows increased activity in MASH in both binding and transporting fatty acids to the mitochondrial matrix. Subsequently, fatty acid beta-oxidation is quite affected. Finally, oxidative phosphorylation is disrupted in both the electron transport chain and ATP synthase parts. This is in agreement with multiple reports of mitochondrial dysregulation and oxidative stress playing a substantial part in MASH\({}^{1940}\).

Compromised mitochondrial functions and bioenergetics in MASH lead to disruption of fatty acid beta-oxidation and oxidative stress

As we observed substantial alterations related to mitochondrial functions, we performed two functional assays focusing on mitochondrial bioenergetics and fatty acid oxidation in order to confirm our findings. Mitochondria isolated from the livers of MASH mice showed reduced complex1 and II-driven respiration as well as maximum respiration. Complex IV was the most affected in MASH livers as shown by its independent activity measurement using the artificial substrates (TMPD+ascorbate) (Fig. 4a). Since pyruvate is the terminal glycolysis product that enters mitochondria via the mitochondrial pyruvate complex (MPC), we examined whether the expression of MPC was affected in the MASH mouse liver. In the immunoblotting assessment, both subunits of MPC were downregulated in MASH mitochondria, indicating that not only the activity of the electron transport chain (ETC) complexes was decreased significantly, but also pyruvate entry into mitochondria was lowered in MASH livers compared to control (Fig. 4b).

We additionally investigated whether fatty acid oxidation was affected in the MASH mouse livers. Respiratory provided a suitable tool to measure fatty acid oxidation since complete oxidation of a fatty acid ends in ATP generation and hence oxygen consumption via the ETC. Palinvolt-carnitine was used as a substrate whereas malate served as a counter ion. As expected, mitochondria isolated from the liver of MASH mice had lower fatty acid oxidation compared to control mitochondria (Fig. 4c). Fatty acid uptake into the mitochondria occurs via the carnitine shuttle. Carnitineine palmitvolttransferase (CPT1)-1 in the outer mitochondrial membrane binds carnitine to the fatty acids to enable entry into the intermembrane space. Subsequently, CPT-2, which is located in the inner mitochondrial membrane, removes carnitine and leads the fatty acid to its oxidation in the mitochondrial matrix. Therefore, we assessed whether the expression of CPT1s was affected in the MASH livers. Although the expression of CPT1a remained unchanged, CPT2 expression was reduced significantly in the MASH livers (Fig. 4d). It is interesting to note that while we confirmed a significant reduction in fatty acid beta-oxidation in the mitochondrial assay, TDMPA had predicted an increase of the pathway activity. This is due to the fact that TDMPA is a direct mapping of transcription-level changes and it does not always correspond to actual changes on the enzymatic and flux levels. However, the goal of TDMPA is to provide a first estimation of those changes and, most importantly, to direct our focus to the most affected metabolic pathways. In this respect, TDMPA correctly predicted major disruption in the mitochondrial metabolism during MASH. The occasional mismatch of the direction of change for each reaction can be attributed to cellular dynamics and molecular organization that cannot be captured solely

Figure 4: **Effect of WD \(+\) CCI on (I) mitochondrial bioenergetics and (II) fatty acid oxidation.** A high-resolution respiratory of oxygen consumption (O2 flux) in mitochondria in liver homogenates of control vs. WD+CC4 group. **I** Immunoblot showing expression of mitochondrial pyruvate carrier MPC1 and MPC2 in liver homogenates. Vinculin served as the loading control. Both MPC1 and MPC2 were lower in WD+CC4 vs. the control group. (unpaired \(t\)-test; \(\lx@math@degree\)\(p\)\(<\) 0.05; 

by the transcriptome and thus needs to be accompanied by assessing enzymatic activities and functional assays.

### Metabolomics and lipidomics confirm observed metabolic alterations in MASH

We next evaluated the accuracy of TDMPA results to guide our investigation of the metabolic alteration occurring in MASH. We performed metabolic profiling within the targeted pathways and identified and quantified over 500 metabolites in mouse liver tissues (see Methods). In total, 252 metabolites were significantly different in the liver between control and MASH (Supplementary Data 1). Specifically, a large number of lipids were significantly different between the control and MASH groups (Fig. 5a, b). Acylarunites were decreased significantly in the MASH group compared to control. This is in agreement with the above-mentioned alterations in the mitochondrial beta-oxidation pathway and reduction in mitochondrial CPT2 expression. Conversely, a large number of TGs, diacylglycer correlates (DGs), monoacylglycer correlates (MGs), phosphatidylethanolamine (PEs), and cholesterol esters (CEs) were increased in MASH. Most bile acids (BAs) and lysophospholipids (LPLs) were also increased significantly in MASH. However, phosphatidylcholines (PCs) exhibited a mixed behavior. The most changes were observed in TGs, LPLs, phospholipids (PLs) (especially PEs), and acylaramines (Fig. 5b, c). Furthermore, we observed alterations of conjugated bile acids including taurondedeoxycholic acid (TCDCA), taurodeoxycholic acid-sulfate (TCDCA-S), tauroduc acid 3-sulfate (TCA-S), taurodeoxycholic acid (TDCA), and tauro-beta-muricholic acid (TMCA). The majority of the conjugated bile acids were significantly increased in MASH, which supports the TDMPA prediction of perturbed bile acid pathways (Fig. 2a).

In accordance with the changes of significant lipids in MASH, we further assessed the fatty acyl (FA) chain length composition in hepatic TGs, DGs, MGs, CEs, PLs, and LPLs (Fig. 6). The majority of altered esterified FAs in MASH were saturated and monounsaturated fatty acids such as FA 18:1, FA 16:0, and FA 16:1 (Fig. 6a). Specifically, oleic acid (FA 18:1) was increased in all detected lipid species. FA 20:3, FA 20:4, FA 22:4, and FA 22:5 within PLs were also increased significantly (Fig. 6b). These fatty acids are produced via desaturation and elongation of linoleic acid. These findings support the prediction of upregulated FA desaturation and FA elongation using the TDMPA approach.

## Discussion

MASH is one of the most prevalent liver diseases worldwide. Despite several developments in animal models, an animal model that resembles the relevant pathophysiology and metabolism needs to be established to better identify targets for treatment and drug testing. Several mouse models of MASH and their histological characterizations have been studied. However, the extent to which their metabolic alterations resemble those in humans remains poorly described. Further evaluation is needed to assess the resemblance of metabolic alterations in animal models to human conditions.

Currently, developed mouse models include dietary, chemically, and genetically induced MASH models. HFD/WD-induced models exhibit the observed metabolic changes across the spectrum of MASH and MASH but might require many weeks of feeding to achieve the desired histopathology[21]. Chemical intoxication leads to fibrosis much faster; however, those models typically do not result in obesity, steatosis, and insulin resistance[13, 14]. Thus, these models are a good way to study

Figure 5: ** Metabolic changes in MASH vs. healthy mouse liver tissue.****a** PCA plot, **b** volcano plot, **c** heatmap of the most changed lipids. MASH metabolic dysfunctionâ€“associated steatohepatitis, PCA principal component analysis. (MASH \(n=17\), control \(n=9\)).

 physical features of liver damage and fibrosis, but potentially not very helpful to study the disease onset. A combination of WD and hepatotoxins to induce fibrosis provides a good alternative for faster disease development while preserving the phenotype[7]. We hypothesized that this approach could lead to similar metabolic changes to human MASH, thus we used the already-established mouse model to test our hypothesis. In order to evaluate the resemblance of this model to human MASH regarding metabolic alterations, we developed TDMPA using genome-scale metabolic models and transcriptomics data and further confirmed our results using functional assays, lipidomics, and metabolomics analysis. Using TDMPA, we were able to map gene expression changes to metabolic reaction rate changes, and estimate the changes across the metabolic pathways. By evaluating the highest-scoring pathways, we defined the metabolic space where most changes occur in disease development and progression, obtaining valuable insights into the metabolic fingerprint of MASH.

Figure 6: **Fatty acyl chain lengths of significantly altered lipids in MASH vs. healthy mouse liver tissue.** A Distribution per the direction of change and lipid class, **b** Box plots of log2-fold-changes per lipid class. The solid line denotes the median of the distribution and the upper and lower hinges correspond to the first and third quartiles, respectively (Tukey representation). MASH metabolic dysfunction-associated steatohepatitis, TG triacylglyceride, DG diacylglyceroleride, MG mono-acycleride, PL phospholipid, LPL lysophospholipid, CE cholesterol ester. (MASH \(n=17\), control \(n=9\)).

 An essential aspect of metabolic pathway analysis is the ability to attain a certain level of granularity in the interpretation of the results, which is often lacking in gene expression analyses. We calculated the changes per metabolic reaction based on gene associations and then scored the metabolic pathways accordingly. TDMPA for the WD+CC4-induced MASH mouse model revealed that many aspects of lipid metabolism are severely affected, which can be grouped into roughly ten pathway families for further investigation. These include bile acid biosynthesis and recycling, fatty acid beta-oxidation, biosynthesis, and metabolism, cholesterol biosynthesis, metabolism, and esterification, leukotriene and arachidonic acid metabolism, carnitine shuttle, oxidative phosphorylation, phospholipid biosynthesis, sphingolipid biosynthesis, and the metabolism of multiple amino acids. We performed the same analysis for human MASH vs. control (Fig. 2b) and MASH progression (MASH F3 vs. MASLD) datasets (Fig. 2c). We observed that the mouse model resembles the human MASH results exceptionally well when compared to control (Fig. 2a vs. b), but the resemblance was less pronounced when compared to human MASH progression results (Fig. 2a vs. c). We can discern a large overlap between the mouse and human MASH vs. control pathways, but the human MASH progression pathways are quite different from both other cases. It is very important to define well the purpose of using animal models in investigating MASH development or progression. It is worth mentioning that the focus of this study was on metabolic alterations that resemble MASH in humans.

Since we observed multiple changes in mitochondrial function, we performed two assays to evaluate the prediction. We confirmed that mitochondrial respiration and oxidation processes are disturbed in MASH mouse liver. These findings support previous hypotheses that mitochondria are heavily involved in the pathogenesis of MASHD, and could be a key player in its progression and regression\({}^{\text{\text{\textregistered}},\text{\textregistered}}\). It is important to note that TDMPA predictions in terms of the direction of changes might not always reflect actual changes in the enzymatic and flux levels, since TDMPA is a direct mapping of transcription-level changes. Nevertheless, TDMPA's value lies in directing our focus to the most affected metabolic pathways and should be accompanied by enzymatic activity assessment and functional assays. Thus, TDMPA findings were further confirmed by lipidomics and metabolomics of the liver. More than 250 metabolites related to enzymatic reactions within the identified pathway families were significantly different in the MASH liver compared to the control. TGs were shown to be especially increased in MASH, thus further confirming the predicted increase of de novo acylglyceride biosynthesis. The increase of primary bile acids and conjugated bile acids has been reported in a previous study of MASH\({}^{\text{\textregistered}}\). Further assessment of the FA chain length composition in hepatic TG, DGAs, GCs, EEs, PIs, and LPIs showed an increase in saturated and monomustarted fatty acids such as FA 18:1, FA 16:0, and FA 16:1. Specifically, oleic acid (FA 18:1) was increased in all detected lipid species.

These results taken together shed light on metabolic pathways that are commonly changed in human and selected mouse models. Our comprehensive assessment provides additional evidence that WD +CC4-induced MASH is an appropriate mouse model for the study of human MASH pathophysiology and metabolism, confirming metabolic and functional resemblance. While TDMPA is limited by GEM annotation and exclusion of several metabolic processes such as signaling, it demonstratively offers an important level of granularity for the study of metabolic pathways in metabolic disorders and facilitates the direct mapping of changes on the gene expression level to metabolic reactions. Additionally, it is a valuable tool for defining metabolic space and better experimental design for lipidomics and metabolomics approaches in both animal models and humans. Finally, TDMPA can be used for the consistent comparison and evaluation of preclinical models, especially for targeting specific enzymes/ metabolic pathways. Accurate preclinical models and suitable methodologies to study them will enable and facilitate drug discovery and testing, identification of risk factors, development of better treatment strategies such as drug combination, and ultimately reduce the global burden of liver disease.

## Data availability

RNA-seq data files from the present study have been deposited into the Gene Expression Omnibus database (www.ncbi.nlm.nih.gov/geo/) with accession number GSE230639. Other datasets used in the study can be found in their respective original publications as listed in Table 1\({}^{\text{\textregistered},\text{\textregistered},\text{\textregistered},\text{\textregistered}}\). The list of 252 statistically significant (FDR < 0.05) lipids and metabolites can be found in Supplementary Data I. Source data for all figures/graphs are available in Figshare (https://doi.org/10.6084/m.figshare.25134470.v1). All other data is available upon reasonable request from the corresponding author.

## Code availability

All in silico analyses were performed in R Statistical Software (v4.0.2; R Core Team 2020) using in-house codes. The following packages were used for visualization: "ggplot2", "pheatmap", "factoextra", and "ggrepel", and package "psych" was used to calculate Cohen's kappa coefficient. The relevant R scripts are available upon reasonable request from the corresponding author.

## Code availability

All in silico analyses were performed in R Statistical Software (v4.0.2; R Core Team 2020) using in-house codes. The following packages were used for visualization: "ggplot2", "pheatmap", "factoextra", and "ggrepel", and package "psych" was used to calculate Cohen's kappa coefficient. The relevant R scripts are available upon reasonable request from the corresponding author.

## References

* [1] Rinella, M. E. et al. A multi-society Delphi consensus statement on new fatty liver disease nomenclature. _J. Hepatol._**78**, 1966-1986 (2023).
* [2] Bellettani, S. The epidemiology of non-alcoholic fatty liver disease. _Liver Int._**37**, 81-84 (2017).
* [3] Younossi, Z. et al. Global burden of NAFLD and NASH: trends, predictions, risk factors and prevention. _Nat. Rev. Gastroenterol. Hepatol._**15**, 11-20 (2018).
* [4] Palk, J. M. et al. Global burden of NAFLD and chronic liver disease among adolescents and young adults. _Hepatology_**75**, 1204-1217 (2022).
* [5] Riazi, K. et al. The prevalence and incidence of NAFLD worldwide: a systematic review and meta-analysis. _Lancet Gastroenterol. Hepatol._**7**, 851-861 (2022).
* [6] Angulo, P. Nonalcoholic fatty liver disease. _Rev. Gastroenterol. Mex._**70**, 52-56 (2005).
* [7] Brunt, E. M. Nonalcoholic steatohepatitisitis: definition and pathology. _Semin. Liver Dis._**21**, 3-16 (2001).
* [8] Silverman, J. F. et al. Liver pathology in morbidity obese patients with and without diabetes. _Am. J. Gastroenterol._**85**, 1349-1355 (1990).
* [9] Loguercio, C. et al. Non-alcoholic fatty liver disease in an area of southern Italy: main clinical, histological, and pathophysiological aspects. _J. Hepatol._**35**, 568-574 (2001).
* [10] Neuschwandre-Tetri, B. A. & Bacon, B. R. Nonalcoholic steatohepatitis. _Med. Clin. North Am._**80**, 1147-1166 (1996).
* [11] Buzzetti, E., Pinzani, M. & Tsochatzis, E. A. The multiple-hit pathogenesis of non-alcoholic fatty liver disease (NAFLD). _Metabolism_**65**, 1038-1048 (2016).
* [12] Masoodi, M. et al. Metabolomics and lipidomics in NAFLD: biomarkers and non-invasive diagnostic tests. _Nat. Rev. Gastroenterol. Hepatol._**18**, 835-856 (2021).
* [13] Sutti, S. A Albano, E. Adaptive immunity: an emerging player in the progression of NAFLD. _Nat. Rev. Gastroenterol. Hepatol._**17**, 81-92 (2020).
* [14] Geng, Y., Faber, K. N., de Meijer, V. E., Blokzijl, H. & Moshage, H. How does hepatic lipid accumulation lead to hepatotoxicity in non-alcoholic fatty liver disease? _Hepatol. Int._**15**, 21-35 (2021).
* [15] Denk, H., Abuga, P. M. & Zattoukal, K. Animal models of NAFLD from the pathologist's point of view. _Biochim. Biophys. Acta. Mol. Basis Dis._**1865**, 929-942 (2019).

 * [16] Fang, T. et al. Mouse models of nonalcoholic fatty liver disease (NAFLD): pathmechanisms and pharmacotherapies. _Int. J. Biol. Sci._**18**, 5681-5697 (2022).
* [17] Farrell, G. et al. Mouse models of nonalcoholic steatohepatitis: toward optimization of their relevance to human nonalcoholic steatohepatitis. _Hepatology_**69**, 2241-2257 (2019).
* [18] Mells, J. E. et al. Saturated fat and cholesterol are critical to inducing murine metabolic syndrome with robust nonalcoholic steatohepatitis. _J. Nutr. Biochem._**26**, 285-292 (2015).
* [19] Henkel, J. et al. Induction of steatohepatitis (NASH) with insulin resistance in wildtype B6 mice by a western-type diet containing soybean al and cholesterol. _Mol. Med._**23**, 70-82 (2017).
* [20] Jensen, T. et al. Fructose and sugar: a major mediator of iron-alcoholic fatty liver disease. _J. Hepatol._**68**, 1063-1075 (2018).
* [21] Jahn, D., Kircher, S., Hermanns, H. M. & Geier, A. Animal models of NAFLD from a hepatologist's point of view. _Biochim. Biophys. Acta Mol. Basis Dis._**165**, 943-953 (2019).
* [22] Clapper, J. R. et al. Diet-induced mouse model of fatty liver disease and nonalcoholic steatohepatitis's reflecting clinical disease progression and methods of assessment. _Am. J. Physiol. Gastroenter. Liver Physiol._**305**, 6483-6495 (2013).
* [23] Nagarajan, P., Mahesh Kumar, M. J., Venkatesan, R., Majundar, S. S. & Juyal, R. C. genetically modified mouse models for the study of nonalcoholic fatty liver disease. _World. J. Gastroenterol._**18**, 1141-1153 (2012).
* [24] Kanno, K., Tazuma, S. & Chayama, K. AT1A-deficient mice show less severe progression of liver fibrosis induced by CCl(4). _Biochem. Biophys. Res. Commun._**308**, 177-183 (2003).
* [25] Kubota, N. et al. A high-fat diet and multiple administration of carbon tetrachloride induces liver injury and pathological features associated with non-alcoholic steatohepatitis in mice. _Clin. Exp. Pharmacol. Physiol._**40**, 422-430 (2013).
* [26] Fujii, M. et al. A murine model for non-alcoholic steatohepatitis showing evidence of association between diabetes and hepatocellular carcinoma. _Med. Mol. Morphol._**46**, 141-152 (2013).
* [27] Tsuchida, T. et al. A simple diet- and chemical-induced murine NASH model with rapid progression of steatohepatitis, fibrosis and liver cancer. _J. Hepatol._**69**, 385-395 (2018).
* [28] Puri, P. et al. A lipidomic analysis of nonalcoholic fatty liver disease. _Hepatology_**46**, 1081-1090 (2007).
* [29] Subudhi, S. et al. Distinct hepatic gene-expression patterns of NAFLD in patients with obesity. _Hepatol. Commun._**6**, 77-89 (2022).
* [30] Yoneda, M. et al. Gene expression profiling of non-alcoholic steatohepatitis using gene set enrichment analysis. _Hepatol. Res._**38**, 1204-1212 (2008).
* [31] Zheng, J., Wu, H., Zhang, Z. & Yao, S. Dynamic co-expression modular network analysis in nonalcoholic fatty liver disease. _Herodot_**158**, 31 (2021).
* [32] Moylan, C. A. et al. Hepatic gene expression profiles differentiate presymptomatic patients with mild versus severe nonalcoholic fatty liver disease. _Hepatology_**59**, 471-482 (2014).
* [33] Schelenberger, J., Park, J. O., Conrad, T. M. & Palsson, B. O. BIGG: a biochemical genetic and genomic knowledgebase of large scale metabolic reconstructions. _BMC Bioinformatics_**11**, 213 (2010).
* [34] Orth, J. D., Thiele, I. & Palsson, B. O. What is flux balance analysis? _Nat. Biotechnol._**28**, 245-248 (2010).
* [35] Patil, K. R. & Nielsen, J. Uncovering transcriptional regulation of metabolism by using metabolic network topology. _Proc. Natl Acad. Sci. USA_**102**, 2885-2689 (2005).
* [36] Mardinoglu, A. et al. Genome-scale metabolic modelling of hepatocytes reveals serine deficiency in patients with non-alcoholic fatty liver disease. _Nat. Commun._**5**, 3083 (2014).
* [37] Sen, P. et al. Quantitative modeling of human liver reveals dysregulation of cytosynopholipid pathways in nonalcoholic fatty liver disease. _iScience_**25**, 104949 (2022).
* [38] Blazier, A. S. & Papin, J. A. Integration of expression data in genome-scale metabolic network reconstructions. _Front. Physiol._**3**, 299 (2012).
* [39] Machado, D. & Hrergard, M. Systematic evaluation of methods for integration of transcriptomic data into constraint-based models of metabolism. _PLoS Comput. Bio._**10**, e1003580 (2014).
* [40] Alemp, D. et al. BRB-seq: ultra-affordable high-throughput transcriptomics enabled by bulk RNA barcoding and sequencing. _Genome Biol._**20**, 71 (2019).
* [41] Jouffe, C. et al. Perturbed rhythmic activation of signaling pathways in mice deficient for Sterol Carrier Protein 2-dependent diurnal lipid transport and metabolism. _Sci. Rep._**6**, 24631 (2016).
* [42] Jouffe, C. et al. Disruption of the circadian clock component BMAL1 elicits an endocrine adaption impacting on insulin sensitivity and liver disease. _Proc. Natl Acad. Sci. USA_**119**, e2200083119 (2022).
* [43] Christinat, N., Valsesia, A. & Masoodi, M. Untargeted profiling of bile acids and lysophopholipids identifies the lipid signature associated with glycemic outcome in an obese non-diabetic clinical cohort. _Biomolecules_**10**, 1049 (2020).
* [44] Eiden, M. et al. Discovery and validation of temporal patterns involved in human brain ketometabolism in cerebral microvodisk fluids of traumatic brain injury patients. _EBio/Medicine_**44**, 607-617 (2019).
* [45] Herzog, R. et al. Lipid/Borner: a software for consensual cross-platform lipoproteins. _PLoS ONE_**7**, e29851 (2012).
* [46] Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._**15**, 550 (2014).
* [47] Robinson, J. L. et al. An atlas of human metabolism. _Sci. Signal._**13**, eaaz1482 (2020).
* [48] Wang, H. et al. Genome-scale metabolic network reconstruction of model animals as a platform for translational research. _Proc. Natl Acad. Sci. USA_**116**, e2102344118 (2021).
* [49] Raudvere, U. et al. p-Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). _Nucleic Acids Res._**47**, W191-W198 (2019).
* [50] Pandey, V., Hadadi, N. & Hatzimanikatts, V. Enhanced flux prediction by integrating relative expression and relative metabolite abundance into thermodynamically consistent metabolic models. _PLoS Comput. Biol._**15**, e1007036 (2019).
* [51] Fang, X., Wallqvist, A. & Reifman, J. Modeling phenotypic metabolic adaptations of Mycobacterium tuberculosis H37Rv under hypoxia. _PLoS Comput. Biol._**8**, e1002688 (2012).
* [52] Navid, A. & Amaas, E. Genome-level transcription data of Yersinia pets analyzed with a new metabolic constraint-based approach. _BMC Syst. Biol._**6**, 150 (2012).
* [53] Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. _J. R. Stat.Soc. Ser. B (Methodological)_**57**, 289-300 (1995).
* [54] Cohen, J. A coefficient of agreement for nominal scales. _Educ. Psychol. Meas._**20**, 37-46 (1960).
* [55] Landis, J. R. & Koch, G. G. The measurement of observer agreement for categorical data. _Biometrics_**38**, 159-174 (1977).
* [56] Lambert, J. E., Ramos-Roman, M. A., Browning, J. D. & Parks, E. J. Increased den novo biogenesis is a distinct characteristic of individuals with nonalcoholic fatty liver disease. _Gastroenterology_**146**, 726-735 (2014).
* [57] Musso, G., Gambino, R. & Cassader, M. Cholesterol metabolism and the pathogenesis of non-alcoholic steatohepatitis. _Prog. Lipid Res_**52**, 175-191 (2013).
* [58] Min, H. K. et al. Increased hepatic synthesis and dysregulation of cholesterol metabolism is associated with the severity of nonalcoholic fatty liver disease. _Cell Metab._**15**, 665-674 (2012).
* [59] Perez-Carreras, M. et al. Defective hepatic mitochondrial respiratory chain in patients with nonalcoholic steatohepatitis. _Hepatology_**38**, 999-1007 (2003).

 https://doi.org/10.1038/s43856-024-00465-3
* [60] Chitturi, S. & Farrell, G. C. Etiopathogenesis of nonalcoholic steatohepatitis. _Semin. Live Dis._**21**, 27-41 (2001).
* [61] Fromenty, B. & Roden, M. Mitochondrial alterations in fatty liver diseases. _J. Hepatatol._**78**, 415-429 (2023).
* [62] Benegjamo, G. et al. The genetic background shapes the susceptibility to mitochondrial dysfunction and NASH progression. _J. Exp. Med._**220**, e20221738 (2023).
* [63] Jiao, N. et al. Suppressed hepatic bile acid signalling despite elevated production of primary and secondary bile acids in NAFLD. G_ut **67**, 1881-1891 (2018).
* [64] Govaere, O. et al. Transcriptome profiling across the nonalcoholic fatty liver disease spectrum reveals gene signatures for steatohepatitis and fibrosis. _Sci. Transl. Med._**12**, eaba4448 (2020).
* [65] Teufel, A. et al. Comparison of gene expression patterns between mouse models of nonalcoholic fatty liver disease and liver tissues from patients. _Gastroenterology_**151**, 513-525 e510 (2016).
* [66] Holland, C. H. et al. Transcriptome cross-species analysis of chronic liver disease reveals consistent regulation between humans and mice. _Hepatol. Commun._**6**, 161-177 (2022).
* [67] Xiong, X. et al. Landscape of intercellular crosstalk in healthy and NASH liver revealed by single-cell exectrome gene analysis. _Mol. Cell_**75**, 644-660 e645 (2019).
* [68] Xiong, X. et al. Mapping the molecular signatures of diet-induced NASH and its regulation by the hepatokine Tsukushi. _Mol. Metab._**20**, 128-137 (2019).
* [69] Blagotnese Cokan, K. et al. Common transcriptional program of liver fibrosis in mouse genetic models and humans. _Int. J. Mol. Sci._**22**, 832 (2021).

## Acknowledgements

The authors would like to thank Philipp Kellman and David Bolet for the technical support with the animal experiments, Regula Thewiltat for the lab assistance, and Dr. Matteo Montari for performing the animal histopathology scoring. M.M. received funding from the Swiss National Science Foundation (SNSF) (grant no. 190686, 213362, and 220325). JFD received funding from the Swiss National Science Foundation (SNSF) (grant no. 310030_185219).

## Author contributions

S.T. and M.M. conceived the idea and planned the study. S.T. designed the method and performed in silico analyses. M.M., J.F.D., and M.S.P.

## 10 Sofia Tsouka1

Footnote 1: Institute of Clinical Chemistry, Inselpital, Bern University Hospital, Bern, Switzerland. 2Department for BioMedical Research, Visceral Surgery and Medicine, University of Bern, Bern, Switzerland. 3Present address: Centre des Maladie Digestwes, Lausanne, Switzerland. 4These authors contributed equally; Pavlra Kumar, Patcharamon Seubnoch. 5e-mail: mojgan.masood@nsei.ch

## References

* [1] S. T. and M. M. M. conceived the idea and planned the study. S.T. designed the method and performed in silico analyses. M.M., J.F.D., and M.S.P.

## References

* [1] S. T. and M. M. M. conceived the idea and planned the study. S.T. designed the method and performed in silico analyses. M.M., J.F.D., and M.S.P.

## 11 Sofia Tsouka1
 